tests/integration_tests/test_scene_r2_8_1.py

```diff
@@ -0,0 +1,136 @@
+"""
+    DSNS 237
+        R2.8.1 Filter out ls8 l1 scenes if the dataset has a child,
+        the child is interim and there is no ancillary
+"""
+
+from pathlib import Path
+import os
+import json
+from click.testing import CliRunner
+import pytest
+from scene_select.ard_scene_select import scene_select, GEN_LOG_FILE
+from scene_select.do_ard import ODC_FILTERED_FILE
+
+from util import (
+    get_list_from_file,
+)
+
+METADATA_DIR = (
+    Path(__file__).parent.joinpath("..", "test_data", "odc_setup", "metadata").resolve()
+)
+METADATA_TYPES = [
+    os.path.join(METADATA_DIR, "eo3_landsat_l1.odc-type.yaml"),
+    os.path.join(METADATA_DIR, "eo3_landsat_ard.odc-type.yaml"),
+]
+
+PRODUCTS_DIR = (
+    Path(__file__).parent.joinpath("..", "test_data", "odc_setup", "eo3").resolve()
+)
+
+PRODUCTS = [
```

Comment:
Only need the ls8 products definitions

Reply:
yep, adjusted, reran tests for this file and it continues to pass :) Ready for review again when you are, Duncan :) Thank you :)